techniques/failures/F94.html

```diff
@@ -59,7 +55,7 @@ <h3>Procedure</h3>
 			<section class="test-results">
 				<h3>Expected Results</h3>
 				<ul>
-					<li>If step #5 is false, then this failure condition applies and the content fails Success Criteria 1.4.4, Resize Text.</li>
+					<li>If the last step is false, then this failure condition applies and the content fails Success Criterion 1.4.4 Resize Text.</li>
```

Comment:
Can a step be false? You just take a  step. Would you not rather say, "if the last check is false" or "if the result of the last check is false"? Maybre I am overthinkign it...

Reply:
while I agree it sounds odd, this wording ("If step X is false" / "If X is false") is used pervasively throughout the techniques (as well as the more correct ("If check X is false"). If we want to tackle this, should probably do it throughout *all* techniques in a separate PR

Reply:
I understand the original wording better.
Nevertheless, if I see correctly none of us 3 is a native speaker.
What do native speakers say?

Reply:
i'd say tackle this in a separate, specific PR that covers *all* techniques, rather than trying to hash it out here in this specific one

Reply:
https://github.com/w3c/wcag/pull/3915

Reply:
agree

Reply:
"die drei deutschsprachigen hier sind sich einig" :)